docs/troubleshooting-guide.md

```diff
@@ -245,7 +264,17 @@ sudo docker logs 'containerName'
 ### I am running a large amount of workflows and MySQL storage disk is full
 To ensure that no data is corrupted for MySQL backed storage for Cromwell, Cromwell on Azure mounts MySQL files on to an Azure Managed Data Disk of size 32G. In case there is a need to increase the size of this data disk, follow instructions [here](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/expand-disks#expand-an-azure-managed-disk).
 
+### Running CWL Workflows on Cromwell on Azure
+Running workflows crafted in the Common Workflow Language(CWL) format is possible with a few modifications to your workflow submission. 
+1) **Ensure your dependencies are accessible by Cromwell** Any additional scripts or subworkflows must be accessible to TES. They can be provided in 3 ways:  
+  * Via a ZIP file in a storage container accessible by Cromwell. **NOTE**- you will **also** have to specify this ZIP file in your [trigger](https://github.com/microsoft/CromwellOnAzure/blob/master/docs/managing-your-workflow.md/#configure-your-cromwell-on-azure-trigger-json-file) file.  
+  * Via public URLs  
+  * Via storage accounts accessbile to TES using either the /storageaccount/container/blob... notation OR the https://storageaccountname.blob.core.windows.net/container/blob format. 
```

Comment:
spelling: accessible. Add link to section in this file #use-input-data-files-from-an-existing-storage-account-that-my-lab-or-team-is-currently-using

Reply:
add a dot at the end of the first sentence (which is in bold)? the same for items 2 and 3 below.